modules/virt-collecting-data-about-vms.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events_monitoring/virt-collecting-virt-data.adoc
+
+[id="virt-collecting-data-about-vms_{context}"]
+= Collecting data about virtual machines
+
+Collecting data about malfunctioning virtual machines (VMs) minimizes the time required to analyze and determine the root cause.
+
+.Prerequisites
+
+* Windows VMs:
+** Record the Windows patch update details for Red Hat Support.
+** Install the latest version of the VirtIO drivers. The VirtIO drivers include the QEMU guest agent.
+** If Remote Desktop Protocol (RDP) is enabled, try to connect to the VMs with RDP to determine whether there is a problem with the connection software.
+
+.Procedure
+
+. Collect `must-gather` data for VMs.
+. Collect screenshots of VMs if they have crashed and before they are restarted.
+. Record common factors, for example, the same host or network.
```

Comment:
I think this should be expanded a bit for clarity (and perhaps translation). This isn't necessarily the best phrasing, but something like: "Record factors that the malfunctioning VMs have in common. For example, the VMs have the same host or network."

Reply:
Probably outside the scope of this PR. I see "for example" as parenthetical, like "such as". Starting a sentence with "For example" sounds like a sentence fragment, although in this case it isn't because there is a a subject and predicate. Maybe I'm confusing it with sentences such as "For example, VMs on the same host or network." Your suggestion makes sense, so I will use it.